galleries/users_explain/quick_start.py

```diff
@@ -13,9 +16,6 @@
 
 """
 
-import matplotlib.pyplot as plt
-import numpy as np
```

Comment:
This is input for a sphinx-gallery based tutorial. The module docstring provides the tutorial heading and imports should not be above that.

Reply:
Resolved